R/shiny.R

```diff
@@ -1065,8 +1059,10 @@ ShinySession <- R6Class(
       }
       # ..stacktraceon matches with the top-level ..stacktraceoff..
       withReactiveDomain(self, {
+        # TODO: should global callbacks be invoked before or after session specific ones?
```

Comment:
I think global callbacks should be invoked after, sort of like how `options(error = <...>)` is your "last-chance" error handler.

For RStudio at least, the goal is to have some mechanism to stop the running application once a session has ended.